gamemodes/terrortown/gamemode/shared/sh_init.lua

```diff
@@ -628,23 +628,30 @@ end)
 ---
 -- Checks whether an equipment is buyable
 -- @param table tbl equipment table
--- @param string team @{ROLE}'s team
+-- @param @{Player} player
 -- @return boolean
 -- @return string text as an icon
 -- @return string result or error
 -- @realm shared
-function EquipmentIsBuyable(tbl, team)
-	if not tbl then
+function EquipmentIsBuyable(tbl, ply)
```

Comment:
okay I see why you changed it above. Isn't this a breaking change? I haven't used this function and I suspect nobody else does. But who knows?

Reply:
It is potentially breaking, I don't think anyone uses it though (It isn't used in any TTT2 repo). I can make it compatible, if necessary

Reply:
It's okay by me